error ngrok - platform darwinarm64 is not supported.

on apple M1 arm64, I just get 
```
Need to install the following packages:
  ngrok
Ok to proceed? (y) y
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! code 1
npm ERR! path /Users/nilleb/.npm/_npx/094a17e86d981b10/node_modules/ngrok
npm ERR! command failed
npm ERR! command sh -c node ./postinstall.js
npm ERR! ngrok - platform darwinarm64 is not supported.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nilleb/.npm/_logs/2020-12-04T22_19_02_541Z-debug.log
```

--- a/download.js
+++ b/download.js
@@ -35,6 +35,7 @@ function downloadNgrok(callback, options) {
     const cdnFiles = {
       darwinia32: cdn + cdnPath + 'darwin-386.zip',
       darwinx64: cdn + cdnPath + 'darwin-amd64.zip',
+      darwinarm64: cdn + cdnPath + 'darwin-amd64.zip',
       linuxarm: cdn + cdnPath + 'linux-arm.zip',
       linuxarm64: cdn + cdnPath + 'linux-arm64.zip',
       androidarm: cdn + cdnPath + 'linux-arm.zip',

